src/platform/tests/TestPlatformTime.cpp

```diff
@@ -72,6 +72,9 @@ static const struct time_test_vector test_vector_system_time_us[] = {
 
 void test_os_sleep_ms(uint64_t millisecs)
```

Comment:
could we write this to call test_os_sleep_us instead? It seems they are the same code. This would avoid a `#idfef`. Ideally I would move this to some 'test_time_operations.h/cpp' however that may be overkill. Why do we even sleep in tests? that is slow :/

Reply:
Yes, please, re having this call sleep_us.